javascript/javascript2/week3/fastestpress.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="fastestpress.css">
+        <title>The fastest presser in this realm game</title>
+</head>
+    <body>
+
+        <h1 class="centeralign">Fastest Presser</h1>
+        <div class="centeralign">
+        <input type="number" id="duration">
```

Comment:
you can add here another attribute to prevent to choose negative numbers
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number